src/index.html

```diff
@@ -22,6 +22,454 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="navigation">
+      <div class="navigation__wraper">
+        <div class="navigation__logo">
+          <img
+            src="./images/logo.png"
+            alt="logo"
```

Comment:
For logos, use the format: "[Organization name] logo." 
Example: 

![image](https://github.com/mate-academy/layout_catalog/assets/54860641/a8f915cd-0fb0-442d-9bd1-727875c20a08)

Reply:
Also need to add a link for the logo.